src/libraries/Microsoft.PowerFx.Interpreter/Functions/CommonErrors.cs

```diff
@@ -108,6 +108,16 @@ public static ErrorValue GenericInvalidArgument(IRContext irContext)
             });
         }
 
+        public static ErrorValue GenericInvalidArgument(IRContext irContext, string message)
+        {
+            return new ErrorValue(irContext, new ExpressionError()
+            {
+                Message = message,
+                Span = irContext.SourceContext,
+                Kind = ErrorKind.InvalidArgument
+            });
+        }
+
```

Comment:
Consider merging this method with the one above (making the message parameter have a default value)

Reply:
Is there a practical technical again? Or just pattern?

Reply:
Less code is almost always better... If we want to change this error kind in the future, for example, we would only have to update one place instead of 2 if it was a single method.